recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ "${build_platform}" == linux-* ]]; then
+if [[ "${target_platform}" == linux-* ]]; then
     export OPENSSL_DIR=$PREFIX
 fi
```

Comment:
actually, let's remove this completely and add `pkg-config` to the build dependencies instead

Reply:
I remember that to not work, but I'm not sure why or if my memory is correct.

Reply:
when i checked last time at one other project, adding `pkg-config` fixed it as well